app/commands/lock_command.rb

```diff
@@ -9,8 +9,17 @@ def run
       return Slash.reply(ValidationErrorMessage.build(record: env)) if env.invalid?
 
       begin
+        previous_owner = locker(env)
         resp = slashdeploy.lock_environment(user, env, message: params['message'].try(:strip), force: params['force'])
         if resp
+          if previous_owner
```

Comment:
Can you move this to the `lock_environment` method in [SlashDeploy::Service](https://github.com/remind101/slashdeploy/blob/1e7d513cb0135e8a52ab21521781e36c78d5cf43/lib/slashdeploy/service.rb#L114-L131)?